pkg/Makefile.am

```diff
@@ -27,20 +27,11 @@ EXTRA_DIST += \
 
 include pkg/build
 
-INSTALL_DATA_LOCAL_TARGETS += install-webpack
-install-webpack: $(WEBPACK_INSTALL) $(MANIFESTS)
-	mkdir -p $(DESTDIR)$(pkgdatadir)
-	$(V_TAR) tar --format=posix -cf - $^ | tar --no-same-owner -C $(DESTDIR)$(pkgdatadir) --strip-components=1 -xvf -
-
-INSTALL_DATA_LOCAL_TARGETS += install-webpack-map
-install-webpack-map: $(wildcard $(top_srcdir)/dist/*/*.map)
-	$(V_TAR) tar --format=posix -c --transform="s@.*dist/@$(debugdir)$(pkgdatadir)/@" -T/dev/null $^ | tar --no-same-owner -C $(DESTDIR)/ -xv
-
-UNINSTALL_LOCAL_TARGETS += uninstall-webpack
-uninstall-webpack:
-	test ! -d $(DESTDIR)$(pkgdatadir) || \
-	  (find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type f -exec rm -f {} \; && \
-	   find $(DESTDIR)$(pkgdatadir) -ignore_readdir_race -type d -empty -delete)
-	test ! -d $(DESTDIR)$(debugdir)$(pkgdatadir) || \
-	  (find $(DESTDIR)$(debugdir)$(pkgdatadir) -ignore_readdir_race -type f -delete && \
-	   find $(DESTDIR)$(debugdir)$(pkgdatadir) -ignore_readdir_race -type d -empty -delete)
+# automake doesn't support stripping initial path components (ie: `dist`) with
+# `nobase_`, so we create a cockpit/ symlink and use that instead.
+# That way, the files land in $(datadir)/cockpit/ and `dist` is avoided.
+all-local:: cockpit
+cockpit:
+	$(AM_V_GEN) ln -sf $(srcdir)/dist $@
```

Comment:
No veto, but it would seem easier and cleaner to just include the symlink into git. This would also make your `$XDG_DATA_DIRS` scenario actually work -- we don't bother with `make` any more when developing in pkg/, it's all webpack-watch and friends now.

Reply:
Sure.

Reply:
Not doing this, because of the automake tar issue.  For the record: creating a tar of the dist *must* dereference symlinks because of the symlinks for `tools/missing` and friends.